### PR TITLE
Feature/school 69/organization

### PR DESCRIPTION
**Pull request by zavarin-michael**

### CODE REVIEW

File: open_schools_platform/api/swagger_tags.py [hunk 1]
```diff
@@ -1,2 +1,4 @@
 user_management_users = "User management. Users"
 user_management_auth = "User management. Authentication"
+organization_management_organizations = "Organization management. Organizations"
+organization_management_employees = "Organization management. Employees"
```
Comment (IlyaBat9) [inline open_schools_platform/api/swagger_tags.py:4]: If this is constant, then CAPITALIZE it.

File: open_schools_platform/common/constants.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class CommonConstants:
+    GOOGLE_API_KEY = os.environ.get("GOOGLE_API_KEY")
+    FIREBASE_URL_TO_GET_SESSION = r"https://www.googleapis.com/identitytoolkit" \
+                                  r"/v3/relyingparty/sendVerificationCode?key="
+    FIREBASE_URL_TO_CHECK_OTP = r"https://www.googleapis.com/identitytoolkit/v3" \
+                                r"/relyingparty/verifyPhoneNumber?key="
+    SMS_PROVIDER_URL = r"https://sms.ru/sms/send"
+    SMS_API_KEY = r"1B525A5E-C342-5299-70CD-F5A0094BC988"
```
Comment (IlyaBat9) [inline open_schools_platform/common/constants.py:14]: DELETE THIS!

**Review by IlyaBat9 — CHANGES_REQUESTED** [review]

File: open_schools_platform/user_management/users/services.py [hunk 3]
```diff
@@ -23,7 +23,7 @@ def create_token(phone: str, session: str) -> CreationToken:
     return token
 
 
-def create_user(phone: str, password: str, name: str, is_active: bool = True, is_admin: bool = False) -> User:
+def create_user(phone: str, password: str, name: str = "", is_active: bool = True, is_admin: bool = False) -> User:
```
Comment (IlyaBat9) [inline open_schools_platform/user_management/users/services.py:26]: We do not pass create user with empty name.

File: open_schools_platform/utils/sms_provider_requests.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+import requests
+
+from open_schools_platform.common.constants import CommonConstants
+
+
+def prepare_phone_number(phone: str):
+    phone = str(phone)  # PhoneNumber cast
+
+    if phone[0] == '+':
+        return phone[1:len(phone)]
```
Comment (IlyaBat9) [inline open_schools_platform/utils/sms_provider_requests.py:12]: What's about Uzbekistan +998?

**Review by IlyaBat9 — CHANGES_REQUESTED** [review]
Fine try.

File: open_schools_platform/utils/sms_provider_requests.py (hunk 4 above)
Comment (IlyaBat9) [reply open_schools_platform/utils/sms_provider_requests.py:12]: Look, @JENIA21 !

File: open_schools_platform/organization_management/employees/services.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+from open_schools_platform.organization_management.employees.selectors import get_employee
+from open_schools_platform.organization_management.organizations.constants import OrganizationConstants
+from open_schools_platform.organization_management.organizations.models import Organization
+from open_schools_platform.user_management.users.models import User
+from open_schools_platform.user_management.users.selectors import get_user
+from open_schools_platform.user_management.users.services import create_user, generate_user_password
+from open_schools_platform.utils.sms_provider_requests import send_sms
+
+
+def create_employee(user: User, organization: Organization, position: str = "", name: str = "") -> Employee:
```
[the remark this answers is not in the document]
Comment (zavarin-michael) [reply open_schools_platform/organization_management/employees/services.py:13]: Defaul values for employees

File: open_schools_platform/user_management/authentication/apis.py [hunk 6]
```diff
@@ -17,7 +17,8 @@
 
 from open_schools_platform.user_management.users.selectors import user_get_login_data
 
-from open_schools_platform.api.swagger_tags import user_management_auth
+from open_schools_platform.api.swagger_tags import SwaggerTags
+from ..users.serializers import UserSerializer
```
[the remark this answers is not in the document]
Comment (zavarin-michael) [reply open_schools_platform/user_management/authentication/apis.py:21]: It generates automatically

File: open_schools_platform/errors/apis.py [hunk 7]
```diff
@@ -1,15 +1,20 @@
...
 from open_schools_platform.errors.services import trigger_errors
 
 
 class TriggerApiException(APIView):
+    @swagger_auto_schema(
+        tags=[SwaggerTags.Errors]
+    )
     def get(self, request):
         data = {
             "drf_default_with_modifications": trigger_errors(drf_default_with_modifications_exception_handler),
```
Comment (IlyaBat9) [inline open_schools_platform/errors/apis.py:20]: Delete hacksoft?

File: open_schools_platform/user_management/users/constants.py [hunk 8]
```diff
@@ -1,17 +1,11 @@
...
-    FIREBASE_URL_TO_CHECK_OTP = r"https://www.googleapis.com/identitytoolkit/v3" \
-                                r"/relyingparty/verifyPhoneNumber?key="
-    GOOGLE_API_KEY = os.environ.get("GOOGLE_API_KEY")
 
-    if not GOOGLE_API_KEY or not is_google_api_key_valid(FIREBASE_URL_TO_GET_SESSION, GOOGLE_API_KEY):
-        warnings.warn("google api key is not valid or is not defined")
+
+class GenerateConstants:
+    Alphabet = string.ascii_letters + string.digits
+    Password_length = 10
```
Comment (IlyaBat9) [inline open_schools_platform/user_management/users/constants.py:11]: CAPITALIZE (

**Review by IlyaBat9 — CHANGES_REQUESTED** [review]
No(

**Review by IlyaBat9 — APPROVED** [review]
Great!

File: open_schools_platform/utils/sms_provider_requests.py (hunk 4 above)
Comment (JENIA21) [reply open_schools_platform/utils/sms_provider_requests.py:13]: why do we need an extra variable in the code (phone)?
And I'd rather use regulars

**Pull request merged**